rad/src/lib.rs

```diff
@@ -51,6 +51,15 @@ pub fn current_active_wips() -> ActiveWips {
     }
 }
 
+/// Auxiliary function that returns the current active wips and the WIPs in voting process as actived
+/// It is only used for testing
+pub fn all_wips_active() -> ActiveWips {
+    let mut h = current_active_wips();
```

Comment:
Why `h`? :rofl:

Reply:
It stands for `HashMap`, because the first version of `ActiveWips` was just a `HashMap`.